templates/element/variables_panel.php

```diff
@@ -24,6 +24,14 @@
     printf('<p class="warning">%s</p>', $error);
 endif;
 
+if (isset($varsMaxDepth)) {
+    $msg = sprintf(__d('debug_kit', '%s levels of nested data shown.'), $varsMaxDepth);
+    $msg .= __d('debug_kit', 'You can overwrite this via the config key');
+    $msg .= ' <strong>DebugKit.variablesPanelMaxDepth</strong>';
+    $msg .= __d('debug_kit', ' | Increasing the depth value can lead to an out of memory error.');
```

Comment:
Having the warning on a new line instead would provide better visibility/readability.

Reply:
So like that?

![image](https://user-images.githubusercontent.com/9105243/143781797-ae284a7b-df2b-4954-b5b4-34e149bf9d53.png)

Reply:
That takes up lot of space. Perhaps keep it in the `info` itself using `<br>`. Also a space is need before `You can...`.

Reply:
I would have to adjust the styling of the info class because it is currently not meant to have multiline text.

![image](https://user-images.githubusercontent.com/9105243/143781972-1f23a9fe-cb5c-45d5-b265-ee6d9ce7c630.png)

Reply:
Pretty please :)

Reply:
Would look like this and will adjust accordingly to how many lines are present.

![image](https://user-images.githubusercontent.com/9105243/143782225-79086fe2-bd90-4d3d-b119-e8db773b05a1.png)

Reply:
![image](https://user-images.githubusercontent.com/9105243/143782403-4324b298-3e04-46d5-b570-7f8fe4fe4467.png)